media/css/cms/flare-footer.css

```diff
@@ -0,0 +1,693 @@
+.fl-footer-secondary a:visited:not(.fl-button) {
+    color: var(--token-neutrals-grey-4);
+}
+
+.whats-new-page .fl-footer-secondary a:link:not(.fl-button),
+.whats-new-page .fl-footer-secondary a:visited:not(.fl-button) {
+    color: var(--fl-theme-color-text-secondary);
+}
+
+.fl-footer-secondary a:hover,
+.fl-footer-secondary a:focus,
+.fl-footer-secondary a:active {
+    text-decoration: none;
+}
+
+/* Legal Links */
+.fl-footer-legal {
+    margin-bottom: calc(var(--token-scale-24) * 1px);
+}
+
+.fl-footer-terms {
+    max-width: 100%;
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+    display: flex;
+    flex-wrap: wrap;
+    gap: calc(var(--token-scale-8) * 1px) calc(var(--token-scale-32) * 1px);
+}
+
+.fl-footer-terms li {
+    display: inline;
+}
+
+.fl-footer-terms a {
+    font-size: calc(var(--token-scale-16) * 1px);
+    line-height: 1.5;
+}
+
+.fl-footer-terms:not(.fl-button),
+.fl-footer-terms a:visited:not(.fl-button) {
+    color: var(--token-neutrals-grey-3);
+    text-decoration: none;
+}
+
+.whats-new-page .fl-footer-terms:not(.fl-button),
+.whats-new-page .fl-footer-terms a:visited:not(.fl-button) {
+    color: var(--fl-theme-color-text-secondary);
+}
+
+.fl-footer-terms a:link:not(.fl-button):hover,
+.fl-footer-terms a:link:not(.fl-button):focus,
+.fl-footer-terms a:visited:not(.fl-button):hover,
+.fl-footer-terms a:visited:focus {
+    text-decoration: underline;
+}
+
+@media screen and (max-width: 767px) {
+    .fl-footer-terms li {
+        display: block;
+        width: 100%;
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+
+/* More Section (Branding + Language) */
+
+.fl-footer-more {
+    padding-bottom: 0;
+    padding-top: calc(var(--token-scale-48) * 1px);
+}
+
+@media screen and (max-width: 767px) {
+    .fl-footer-more {
+        border-top: 1px solid var(--token-neutrals-grey-3);
+    }
+
+    .whats-new-page .fl-footer-more {
+        border-color: var(--fl-theme-color-text-secondary);
+    }
+}
+
+@media screen and (min-width: 768px) {
+    @supports (display: flex) {
+        .fl-footer-more {
+            display: flex;
+            justify-content: space-between;
+            align-items: flex-end;
+        }
+    }
+}
+
+.fl-footer-branding {
+    margin-bottom: calc(var(--token-scale-32) * 1px);
+}
+
+@media screen and (min-width: 768px) {
+    .fl-footer-branding {
+        margin-bottom: 0;
+    }
+}
+
+.fl-footer-branding h2 {
+    font-size: calc(var(--token-scale-24) * 1px);
+    line-height: 1.3;
+    padding-top: 0;
+    color: var(--token-neutrals-white);
+    font-weight: bold;
+}
+
+.whats-new-page .fl-footer-branding h2 {
+    color: var(--fl-theme-color-text-body);
+}
+
+.fl-footer-branding p {
+    margin: 0;
+    font-size: calc(var(--token-scale-16) * 1px);
+    color: var(--token-neutrals-white);
+    line-height: 1.5;
+}
+
+.whats-new-page .fl-footer-branding p {
+    color: var(--fl-theme-color-text-body);
+}
```

Comment:
I don't see the class `.fl-footer-branding` `.fl-footer-more` referenced by any HTML?